Make.config

```diff
@@ -200,9 +200,9 @@ MACCATALYST_NUGET_VERSION_NO_METADATA=$(MACCATALYST_NUGET_VERSION)$(NUGET_PREREL
 MACCATALYST_NUGET_VERSION_FULL=$(MACCATALYST_NUGET_VERSION_NO_METADATA)+$(NUGET_BUILD_METADATA)
 
 # Xcode version should have both a major and a minor version (even if the minor version is 0)
-XCODE_VERSION=14.3.1
-XCODE_URL=https://dl.internalx.com/internal-files/xcodes/Xcode_14.3.1.xip
-XCODE_DEVELOPER_ROOT=/Applications/Xcode_14.3.1.app/Contents/Developer
+XCODE_VERSION=15.0
+XCODE_URL=https://dl.internalx.com/internal-files/xcodes/Xcode_15_beta_4.xip
```

Comment:
Not going straight for beta 5? 😅

Reply:
Let me get Beta 4 in a decent place first 😅